### ISSUE TITLE
Method "GetAll" with signature "s" on interface "org.freedesktop.DBus.Properties" doesn't exist

### ISSUE THREAD

**Issue by obadz** [omeid/upower-notify#1]
I've packaged `upower-notify` for NixOS (see https://github.com/obadz/nixpkgs/commit/b3137e4e11375fae4227f9b083762405b6692939)

I get this error on startup:

```
$ upower-notify 
2016/03/02 13:53:30 Method "GetAll" with signature "s" on interface "org.freedesktop.DBus.Properties" doesn't exist
```

I've got `dbus-daemon` and `upowerd` running. Any ideas?

**Comment by omeid** [comment]
Hi Obadz,

Can you please run the following and report the out please?

``` bash
$  $ dbus-daemon --version
```

and then

``` bash
$ dbus-send --print-reply \
            --system \
            --dest=org.freedesktop.UPower \
            /org/freedesktop/UPower/devices/battery_BAT0 \
            org.freedesktop.DBus.Properties.GetAll \
            string:org.freedesktop.UPower.Device
```

**Comment by obadz** [comment]
Hi @omeid,

Thanks for looking into this.

```
$ dbus-daemon --version
D-Bus Message Bus Daemon 1.8.20
Copyright (C) 2002, 2003 Red Hat, Inc., CodeFactory AB, and others
This is free software; see the source for copying conditions.
There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
$ dbus-send --print-reply \
>             --system \
>             --dest=org.freedesktop.UPower \
>             /org/freedesktop/UPower/devices/battery_BAT0 \
>             org.freedesktop.DBus.Properties.GetAll \
>             string:org.freedesktop.UPower.Device
Error org.freedesktop.DBus.Error.UnknownMethod:
```

Looks like the issue is that my battery is `/org/freedesktop/UPower/devices/battery_BAT1`, not `/org/freedesktop/UPower/devices/battery_BAT0` (same command as above with `BAT1` instead of `BAT0` works).

Is there away to pass the battery's address by command-line argument to `upower-notify`?

**Comment by omeid** [comment]
That makes sense. While you're at it, can you see if 

``` bash
$ dbus-send --print-reply \
            --system \
            --dest=org.freedesktop.UPower \
            /org/freedesktop/UPower/devices/DisplayDevice \
            org.freedesktop.DBus.Properties.GetAll \
            string:org.freedesktop.UPower.Device
```

Works for you too?

**Comment by omeid** [comment]
Closed with #2.
